<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>串口设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>端口：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="port_comboBox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>波特率：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="brud_comboBox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>数据位：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="byte_comboBox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>停止位：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="stop_byte_comboBox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,1">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>校验位：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="check_byte_comboBox"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="open_uart_btn">
          <property name="text">
           <string>打开串口</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="search_uart_btn">
          <property name="text">
           <string>搜索串口</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;串口未连接&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;网络未连接&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTextEdit" name="textEdit"/>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>发送设置</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>发送目标：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="send_select_comboBox"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="send_btn">
              <property name="text">
               <string>发送</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="send_clear_btn">
              <property name="text">
               <string>清空发送</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
